lesson_38/my-project/src/components/Product.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import './Product.css';
+
+export default function Product(props) {
+
+  return (
+    props.products.map((product) =>
```

Comment:
this map should be in `Products` component
and here just an html code of product